README.md

```diff
@@ -716,6 +717,19 @@ __AWESOME__ apps counter: __253__ 🎉
 - [x] [GitHub](https://github.com/Pool-Of-Tears/Myne)
 - [ ] Official page
 
+## Education
+
+### Quiz-Platform
+
+> Quiz Platform is a Open Source knowledge testing platform for Android based on Jetpack Compose and Google solutions.
+
+<img alt="QuizPlatformIcon" height="64" src="https://raw.githubusercontent.com/Yugyd/quiz-platform/master/app/src/main/res/mipmap-hdpi/ic_launcher.webp">
+
+- [x] [Google Play]()
```

Comment:
I thought you had a link to Google Play. Have you forgotten it here in ()?

Reply:
Sorry, fixed it